bloomstack_core/patches/v0_0_1/removed_title_field_from_work_order_doc.py

```diff
@@ -0,0 +1,4 @@
+import frappe
+
+def execute():
+	frappe.delete_doc_if_exists("Work Order", "title")
```

Comment:
```suggestion
	frappe.delete_doc_if_exists("Custom Field", "Work Order-title")
```
- You have to delete the Custom Field.